test/parallel/test-cluster-bind-twice.js

```diff
@@ -18,88 +18,88 @@
 //
 // See https://github.com/joyent/node/issues/2721 for more details.
 
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var fork = require('child_process').fork;
-var http = require('http');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const fork = require('child_process').fork;
+const http = require('http');
 
-var id = process.argv[2];
+const id = process.argv[2];
 
 if (!id) {
-  var a = fork(__filename, ['one']);
-  var b = fork(__filename, ['two']);
+  const a = fork(__filename, ['one']);
+  const b = fork(__filename, ['two']);
 
-  a.on('exit', function(c) {
+  a.on('exit', common.mustCall((c) => {
     if (c) {
       b.send('QUIT');
       throw new Error('A exited with ' + c);
     }
-  });
+  }));
 
-  b.on('exit', function(c) {
+  b.on('exit', common.mustCall((c) => {
     if (c) {
       a.send('QUIT');
       throw new Error('B exited with ' + c);
     }
-  });
+  }));
 
 
-  a.on('message', function(m) {
+  a.on('message', common.mustCall((m) => {
     if (typeof m === 'object') return;
-    assert.equal(m, 'READY');
+    assert.strictEqual(m, 'READY');
     b.send('START');
-  });
+  }));
 
   let ok = false;
 
-  b.on('message', function(m) {
+  b.on('message', common.mustCall((m) => {
     if (typeof m === 'object') return; // ignore system messages
-    assert.equal(m, 'EADDRINUSE');
+    assert.strictEqual(m, 'EADDRINUSE');
     ok = true;
     a.send('QUIT');
     b.send('QUIT');
-  });
+  }));
 
-  process.on('exit', function() {
+  process.on('exit', () => {
     assert(ok);
   });
 } else if (id === 'one') {
   if (cluster.isMaster) return startWorker();
 
-  http.createServer(common.fail).listen(common.PORT, function() {
+  http.createServer(common.fail).listen(common.PORT, common.mustCall(() => {
     process.send('READY');
-  });
+  }));
 
-  process.on('message', function(m) {
+  process.on('message', common.mustCall((m) => {
     if (m === 'QUIT') process.exit();
-  });
+  }));
 } else if (id === 'two') {
   if (cluster.isMaster) return startWorker();
 
   let ok = false;
-  process.on('exit', function() {
+  process.on('exit', () => {
```

Comment:
I don't believe so. The `ok` is only set under certain conditions that are not covered simply by the addition of `common.mustCall()`. For instance, if the `message` event is triggered only because of system messages, the necessary `common.mustCall()` won't be triggered.